### DISCUSSION TITLE
Disable a detector

### DISCUSSION THREAD

**Discussion by vinayaugustine** [gregnavis/active_record_doctor#87] [Q&A] [answered]
Is it possible to disable a detector entirely? We get a lot of issues from the `missing_foreign_keys` detector, but it is not how we have our database structured.

**Comment by gregnavis** [comment, marked as answer]
No, it's currently not possible. I definitely agree it's a desirable feature and will add it to the roadmap.

**Reply by gregnavis** [reply]
@vinayaugustine, I've just released 1.10 which includes the feature you requested. Thank you for your suggestion!

**Comment by gregnavis** [comment]
@vinayaugustine, thank you for the suggestion. It'll be part of the next release: https://github.com/gregnavis/active_record_doctor/commit/0c249b1ffd479c3173f0a11f14305d67da50b6dc.

**Reply by vinayaugustine** [reply]
awesome, thanks much!
